src/form.ts

```diff
@@ -15,7 +15,7 @@ export type FormValue = { [ key: string ]: any };
 function findFirstEnabledIndex(select: HTMLSelectElement) {
 	const length = select.options.length;
 	for (let i = 0; i < length; i++) {
-		if (!select.options[i].disabled) {
+		if (!(<HTMLOptionElement> select.options[i]).disabled) {
```

Comment:
Actually it was an array of `any`, which is why you could have whatever you wanted as a property.  In 1.8 it was more specific to `Element`.  See Microsoft/TSJS-lib-generator#68